calico-enterprise/multicluster/federation/kubeconfig.mdx

```diff
@@ -22,85 +22,106 @@ This how to guide uses the following {{prodname}} features:
+Policy can typically only reference the endpoint identity (e.g. pod labels) of local endpoints. Federated endpoint identity enables local policy rules to reference remote endpoint identities.
 
-In the following example, we federate three clusters with each other: `cluster-a`, `cluster-b`, and `cluster-c`:
+### RemoteClusterConfiguration
+RemoteClusterConfiguration is the resource that configures a local cluster to sync resources from a remote cluster. It primarily describes how a local cluster establishes that connection to the remote cluster through which resources are synced.
 
-- **cluster a**
+The resources synced through this connection enable the local cluster to reference remote endpoint identity and establish cross-cluster overlay routes.
 
-  Create two RemoteClusterConfiguration resources: 1 for cluster-b, 1 for cluster-c
+RemoteClusterConfiguration creates this connection in one direction. If you want identity-aware policy on both sides (i.e. both clusters) of a connection, or you want {{prodname}} to establish cross-cluster overlay networking, you need to create a RemoteClusterConfiguration for both directions.
 
-- **cluster b**
+### kubeconfig files
+Each cluster in the cluster mesh should have a dedicated kubeconfig file used by other clusters in the mesh connect and authenticate.
 
-  Create two RemoteClusterConfiguration resources: 1 for cluster-a, 1 for cluster-c
 
-- **cluster c**
-
-  Create two RemoteClusterConfiguration resources: 1 for cluster-a, 1 for cluster-b
-
-:::note
-
-Although this example applies remote cluster configurations symmetrically across clusters, this is not required; add RemoteClusterConfiguration resources only where needed.
+## Before you begin
+**Required**
+- [Install {{prodname}}](../../getting-started/install-on-clusters/kubernetes/index.mdx)
+- [Ensure pod IP routability](#ensure-pod-ip-routability)
 
-:::
+## How to
+- [Create kubeconfig files](#create-kubeconfig-files)
+- [Create RemoteClusterConfiguration](#create-remoteclusterconfigurations)
+- [Validate federation and multi-cluster networking](#validate-federation-and-multi-cluster-networking)
+- [Create remote-identity-aware network policy](#create-remote-identity-aware-network-policy)
+- [Troubleshoot](#troubleshoot)
+- [Configure IP pool resources](#configure-ip-pool-resources)
 
-## Before you begin
 
-**Supported networking**
+### Ensure pod IP routability
+Federation of workload endpoint identities only has utility when [Pod IP routability](./overview#pod-ip-routability) is established between clusters. If your clusters are using a supported overlay networking mode, {{prodname}} can automatically meet this requirement when clusters are connected.
```

Comment:
I've made the suggested change for the first sentence, but I prefer to how the second sentence is currently phrased. Specifically:
* I prefer the less absolute terminology of "can ... meet" over "is .. handled", since the usage of supported overlay mode alone is not enough
* I think noting that this requirement is met "when clusters are connected" helps the user understand what's to be expected in going through this configuration process

Reply:
I'm fine with that.